src/main/java/com/cst438/controller/EnrollmentController.java

```diff
@@ -13,30 +17,37 @@
 @RestController
 @CrossOrigin(origins = "http://localhost:3000")
 public class EnrollmentController {
+    @Autowired
+    EnrollmentRepository enrollmentRepository;
 
     // instructor downloads student enrollments for a section, ordered by student name
     // user must be instructor for the section
     @GetMapping("/sections/{sectionNo}/enrollments")
     public List<EnrollmentDTO> getEnrollments(
             @PathVariable("sectionNo") int sectionNo ) {
+        //  hint: use enrollment repository findEnrollmentsBySectionNoOrderByStudentName method
+        List<Enrollment> enrollments = enrollmentRepository.findEnrollmentsBySectionNoOrderByStudentName(sectionNo);
+        List<EnrollmentDTO> dtoEnrollments = new ArrayList<>();
```

Comment:
`dtoEnrollments` is unused

Reply:
Fixed via commit [6e88aa2](https://github.com/jdolarcsumb/CST438-Assignment2-02/commit/6e88aa263d74a2cde9030f3e756ae58add45d49f)